### PR TITLE
master added template stuff and initial home page

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+#### Setup
+
+- Clone this repository
+
+
+##### Code
+
+- Create a Configuration table in your (Development) local Azure Storage account.
+- Add a row to the Configuration table with fields: PartitionKey: LOCAL, RowKey: SFA.DAS.RoatpOversite_1.0, Data: {
```
Comment (David-B-Read) [inline README.md:25]: Should be Oversight?

File: src/SFA.DAS.RoatpOversight/SFA.DAS.RoatpOversight.Web/Domain/Roles.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        public const string RoatpGatewayTeam = "APR";
+        public const string RoatpGatewayAssessorTeam = "GAC";
+        public const string RoatpFinancialAssessorTeam = "FHC"; // RoATP FHA
+        public const string RoatpApplicationOversightTeam = "AOV";
+        public const string RoatpOversightTeam = "AAC";
+
+        public static bool HasValidRole(this ClaimsPrincipal user)
+        {
+            return user.IsInRole(RoatpGatewayTeam)
```
Comment (David-B-Read) [inline src/SFA.DAS.RoatpOversight/SFA.DAS.RoatpOversight.Web/Domain/Roles.cs:17]: Should this just be the AOV role?

File: README.md (hunk 1 above)
Comment (MarkFCain) [reply README.md:25]: haha yes it shood

File: src/SFA.DAS.RoatpOversight/SFA.DAS.RoatpOversight.Web/Domain/Roles.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpOversight/SFA.DAS.RoatpOversight.Web/Domain/Roles.cs:17]: not sure at this point.  Certainly the link and subsequent pages should only be AOV, but not sure about the more general pages, such as 'Access Denied'.  I hope that makes sense.  I'll restrict it to AOV for now, and we'll figure it out from there.

**Pull request merged**